.pipelines/clang-format-ci.yml

```diff
@@ -0,0 +1,56 @@
+name: $(Year:yy).$(Month).$(DayOfMonth).$(rev:r)
+
+pr:
+  branches:
+    include:
+    - main
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '14.x'
+  displayName: 'Install Node.js'
+
+- script: |
+    npm install
+  displayName: 'npm install'
+  workingDirectory: source/nodejs
+
+- script: |
+    npm run verify --verbose
+  displayName: 'npm run verify --verbose'
+  workingDirectory: source/nodejs
+
+- bash: |
+    echo "##[command]Three ways to fix the format problem"
+    echo '##[command]1. powershell -ExecutionPolicy Bypass scripts\FormatSource.ps1 -ModifiedOnly $False'
+    echo "##[command]2. or cd source\nodejs, then npm run format --verbse"
+    echo "##[command]3. or download format.patch from pippeline, then git apply format.patch"
+  displayName: 'How to fix the format problem'
+  condition: failed()
+
+- script: |
```

Comment:
nit: wouldn't it be if the script tasks are consolidated to one script task?

Reply:
no. pipeline is enough since we don't need to reuse it